gadget-container/Makefile

```
@@ -2,12 +2,14 @@
 gadget-container-deps: ocihookgadget gadgettracermanager nrigadget
 
 
+TARGET_ARCH ?= $(shell go env GOHOSTARCH)
```

Shouldn't this be 
```suggestion
TARGET_ARCH ?= $(shell go env GOARCH)
```

(not relevant when called from the Dockerfiles, or on usual setups where GOHOSTARCH=GOARCH)